src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/CommitmentOrchestrator.cs

```diff
@@ -250,11 +222,16 @@ public class CommitmentOrchestrator : BaseCommitmentOrchestrator
             });
 
             if (relationshipRequest.Relationship.Verified.HasValue)
```

Comment:
Looks like it could be null, but it's existing code and we need to 'chunk out' the refactoring and there's plenty for this user story, so will leave for another day.